setup.py

```diff
@@ -1,9 +1,9 @@
 from distutils.core import setup, Extension
 from pip.req import parse_requirements
-install_reqs = parse_requirements('requirements.txt', session=False)
+# install_reqs = parse_requirements('requirements.txt', session=False)
```

Comment:
Usually people generate `requirements.txt` *from* setup.py not the other way around.  I'd keep stuff in `install_requires`.

C.f. https://caremad.io/posts/2013/07/setup-vs-requirement/